src/client/modules/content/pages/datatable/CollaboratorDataTable.jsx

```diff
@@ -0,0 +1,296 @@
+import React from 'react';
+import CollaboratorTableEntry from './CollaboratorTableEntry.jsx';
+import DataTableCategory from './DataTableCategory.jsx';
+
+export default class CollaboratorDataTable extends React.Component {
```

Comment:
Have you considered sub-classing the table?
It seems that the pagination and search could be defined on a base class.